### DISCUSSION TITLE
A page dedicated to events

### DISCUSSION THREAD

**Discussion by Nepiki** [RetroAchievements/RAWeb#2840] [Ideas]
This idea has been tossed up a few times, but I wanted to finalize it by providing a direct mock-up, in collaboration with the events team to hear their wishes on what such a page could benefit from. What I chose to go for is essentially a ''duplicated'' front page, but with unique elements to make all information clearly visible. Here is the result I came up with:

![image](https://github.com/user-attachments/assets/797d5249-60e0-49bc-ae9f-d811105f3262)

Now to go in on a few comments:

1. Calendar

Now this is the only part I lazily photoshopped in because my website altering™ skills have a limit, but this was one of the points brought up: an easily visible calendar, that would display what events are going on, and when they are getting an update (e.g. AOTW every monday, Bounty Hunters ever 2 weeks, Leaderboard of the Month on the 1st and 15th). It is complemented by a list of all timed/permanent events under, that has a link to the forum post, a description of what the event is about, and optionally yet again a datespan detailing when the event starts and ends. Optionally, the description of an event could be in a dropdown, depending on how long the page becomes because of them.

2. Event updates

This is basically just a lazy rehash of the regular news, but it's good to have this information be available on the page as well. What has been tossed up by Drisc, is that news posts can be given a tag (e.g. events), making them automatically appear here.

3. Evergreen Events

It has been tossed up for Evergreen Events to also get their own page, but I wanted to include it within this page because I think the space is there for it, by following a table structure similar to new sets/sets in progress.

-------------

There are a few feedback points that are still open:

1. Should there be an additional tab in the top bar with events?
2. Should the AOTW/AOTM widget on the home page remain in-tact, or changed for this page?
3. I used badges for the events, but these are often revealed near the end of the event. Should we put a placeholder image there?

-------------

And those are my thoughts on the Events Page. Personally speaking, I think it would be a massive benefit to the site to have a page like this, as events are "the gateway drug to involvement" (stole this from #1098 , don't mind me), and are also a high contributor in getting people engaged with the community. Additionally, it works great for promotion.

**Comment by wescopeland** [comment]
All for it! I have a few comments to hopefully help guide & focus the discussion:

1. Let's try to avoid getting attached to any mock-ups/concepts for what this page should look like. I think we should focus on desired functionality above all else, then let the UI/UX fall where it may. Reasoning: the dual-column layout may not be a good fit for this page's functionality (it's space-constrained and doesn't play well with mobile). We may want something entirely different.

2. A top bar tab/nav item does make a lot of sense.

3. AOTW should remain intact on the home page. We have a lot of data supporting that this element gets high traction - any change to it should be a data-driven choice and possibly A/B tested.

**Reply by Nepiki** [reply]
Definitely agreed on point 1; I did some quick editing of the homepage to make this mock-up, but it may be more beneficial to use the space better, e.g. give the calendar more space. Functionality is absolutely the first priority, with the mock-up containing everything desired, but not necessarily final.

Also certainly agreed on point 3! I know that has been part of another discussion ( #1098 ), but at this point in time I have no real idea what I would even change besides what is mentioned there, but that it should remain is clear.

**Reply by StingX2** [reply]
> 3. AOTW should remain intact on the home page. We have a lot of data supporting that this element gets high traction - any change to it should be a data-driven choice and possibly A/B tested.

A/B test would show that whatever you put there will literally get the same traction, id bet almost anything.

**Reply by wescopeland** [reply]
My perception is we're talking about removing it from the home page altogether, which I think should be a carefully-made choice.

**Reply by StingX2** [reply]
> My perception is we're talking about removing it from the home page altogether, which I think should be a carefully-made choice.

oh I don't like that Mr Stark, that seems like a mess with success choice.

**Comment by Xeinok-Pals** [comment]
1. I'm in agreement that Events deserves its own main page and its own top bar link. Not trying to be too attached to any specific UI/UX or mockup/concept and all but I do want to say that I see a place for some sort of display that attempts to equalize the design priority of each event. Something like https://www.meetup.com with filters to view various RA Events of different types
![image](https://github.com/user-attachments/assets/7599149d-bf95-41ba-8a0b-c8ea030601b3)

2. No offense to anyone but I find Events to be extremely overwhelming, especially to new users. Some sort of gamified "welcome to RA introductory quest" that guides you through earning your first cheevo, beating/mastering your first game, and having you complete an objective from a few various Events, etc. could be a cool way to onboard new users.

3. Continuing with the "Events are overall overwhelming" thing (and no offense to anyone again) I think the Writing Team should work with the Events Team to help heavily reduce the enormous walls-of-text that almost every Event forum post and Event update seems to become. There has to be a way to make these more succinct, impactful, and easy to auto-translate for our non-English users.

4. I just don't think that carousels have any place in modern web design. Most of these suffer from the same issue of having way too much explainer text and too much non-impactful text overall. It's very easy for your eyes to just glaze over and complete ignore these:
![image](https://github.com/user-attachments/assets/848e61fb-cbbb-4c69-bfc0-29fd693a7273)

**Reply by TrashcityRA** [reply]
2. Solid point. I think a tutorial event would help out loads of people out in understanding how events work out. Could be a neat little starter badge for users.

4  I agree, prime example they showcased the Thanksgiving mini event which was on the front page but then a day later a new article replaced it meaning users would have to stay on the page for it to auto cycle or click on news arrow themselves.

**Comment by TrashcityRA** [comment]
I wanted to say my opinion on events from a user who just started caring about them recently.  

Events seriously is probably one of the coolest thing on this site, but I personally think RA needs to find a way to simplify a way to explain events. When I first started I didn't go for events because they seemed so complicated.  It feels like you have to really understand and decipher what the event really is. 

At the end of the day they are not all that complicated it's just the way RA is presenting it, seems way more harder than it should be. I hope you guys can figure out a way because I know loads of users would do events if you guys found away to present/explain it better on the website.

**Comment by LuneDeCuivre** [comment]
Definitively agree with this. A way to see which events are ongoing would be awesome.
I know someone on discord thought the halloween event would be done today since it was halloween, a page would help them know the info easily.
